### ISSUE TITLE
[AutocompleteInput/Select] - add a way to render something different if menu is empty

### ISSUE THREAD

**Issue by michalkvasnicak** [Byteclaw/visage#520]
Now we have a mechanism to render custom menu. Add default no results or something like this to Select and render nothing in AutocompleteInput.

Basically on Select it could be handled by `emptyResultsText` prop. In Autocomplete this shouldn't render anything because AutocompleteInput should work just like suggester.

**Comment by jurajhrib** [comment]
Maybe a children prop can be used for "default" empty item? Do we want to have option to render totally custom element or component when set is empty?

**Comment by michalkvasnicak** [comment]
Hard to say because you can have an use case for either rendering totally nothing or some default empty item. But for example what would you do if you want to render something like `Create tag "abwab"` when you type `abwab` and there is no result for it.

**Comment by jurajhrib** [comment]
Got it. Maybe we'll need even multiple props for dealing with this. One for "empty" state, when use did not even search for anything, but we want to display something in the autocomplete.

Then for "nothing found" state. And then maybe some boolean flags to control these?

**Comment by michalkvasnicak** [comment]
Yes maybe it could accept the state of the Select/AutocompleteInput and decide what will be visible in the popover. At the moment `menu` prop is completely responsible of rendering the Popover with menu. We could have another prop for empty result I guess (This is not a breaking change as it's completely optional). 

- Nothing found state can be rendered using custom `menu` because it accepts `options`. 
- Empty state on the other hand doesn't have any mechanism so now it's handled by the same `menu` prop.

Empty state could be used only if the prop is set up otherwise `menu` would be used.

What do you mean by empty state on Autocomplete? Do you mean that if you focus an Autocomplete input then you'll automatically show something? Because I'm not sure if this could be handled by internal logic of this component because `open` state is handled internally.
